src/routes/forms.test.js

```diff
@@ -503,10 +495,6 @@ describe('Forms route', () => {
 
       expect(response.statusCode).toEqual(notFoundStatusCode)
       expect(response.headers['content-type']).toContain(jsonContentType)
-      expect(response.result).toMatchObject({
-        error: 'Not Found',
-        message: 'Failed'
-      })
```

Comment:
We should keep these lines too?

Should make sure that errors thrown by S3 are logged but don't make it to the response